main/main.cpp

```diff
@@ -426,6 +426,16 @@ void finalize_theme_db() {
 #define MAIN_PRINT(m_txt)
 #endif
 
+/**
+ * Prints a message related to invalid command line arguments, with a blank line added automatically at the end.
+ * The formatting is similar to `ERR_PRINT()`, but without a visible stack trace.
+ * Unlike `ERR_PRINT()`, this method can be called early in the initialization
+ * without the message disappearing on Windows.
+ */
+void Main::print_argument_error(const char *p_message) {
+	OS::get_singleton()->print("\u001b[1;91mERROR:\u001b[0;91m %s\u001b[0m\n", p_message);
```

Comment:
Note that if I use `printerr()` instead of `print()` here (which we technically should do), the message will fail to appear on Windows (tested in Git Bash in the VS Code integrated terminal). This happens both with the main and `console` executable.

However, it works if I start a cmd session within that Git Bash session.